### ISSUE TITLE
Broken external links on the WCAG 2.2 Understanding and Techniques pages

### ISSUE THREAD

**Issue by TestPartners** [w3c/wcag#4964]
In response to https://github.com/w3c/wcag/issues/4961#issuecomment-3972232103, I have run an automated link checker on the Understanding and Techniques pages for WCAG 2.2. The broken external links are listed below. I opened a [separate issue for the broken internal links](https://github.com/w3c/wcag/issues/4962).

http://webdesign.about.com/od/cookies/a/aa083198.htm
This is linked from the "Write Your First HTTP Cookie" link on https://www.w3.org/WAI/WCAG22/Techniques/css/C29

http://www.accessibilitynews.ca/cwdo/resources/resources.php?resources=187
This is linked from the "Better Connected, Better Results: Alt Text" link on https://www.w3.org/WAI/WCAG22/Understanding/non-text-content

http://www.vischeck.com/daltonize/runDaltonize.php
This is linked from the "Tool to convert images based on color loss..." link on the following four pages:
https://www.w3.org/WAI/WCAG22/Techniques/general/G17
https://www.w3.org/WAI/WCAG22/Techniques/general/G18
https://www.w3.org/WAI/WCAG22/Techniques/general/G145
https://www.w3.org/WAI/WCAG22/Techniques/general/G148

https://www.vischeck.com/daltonize/runDaltonize.php
This is linked from the "Tool to convert images based on color loss..." link on https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum

http://www.xmlforasp.net/SchemaValidator.aspx
This is linked from the "Schema Validator" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G134

https://file000.flaticon.com/downloads/license/license.pdf
This is linked from the "Flaticon Basic License" link on https://www.w3.org/WAI/WCAG22/working-examples/aria-role-status-shoppingcart/

http://sovo.vitac.com/
This is linked from the "Sovo Closed Captioning and Audio Description Services" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G157

http://www.chami.com/html-kit/faq/pages/validate_tools.html
This is linked from the "How do I validate my code or check for possible errors?" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G134

http://www.chami.com/html-kit/faq/pages/w3c_markup_validator.html
This is linked from the "Can I use the W3C MarkUp Validation Service to validate HTML?" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G134

http://www.sign-lang.uni-hamburg.de/signingbooks/deliver/d31/deliv_31_part3-2.html
This is linked from the "Sign Language presentation" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G54 and 
https://www.w3.org/WAI/WCAG22/Techniques/general/G81

http://www.sign-lang.uni-hamburg.de/signingbooks/sbrc/grid/d71/guide12.htm
This is linked from the "chapter 12, “Filming the Signer(s)"" link on the following two pages:
https://www.w3.org/WAI/WCAG22/Techniques/general/G54  
https://www.w3.org/WAI/WCAG22/Techniques/general/G81
It is also linked from the "Techniques for filming sign language interpreters" link on the following two pages:
https://www.w3.org/WAI/WCAG22/Understanding/sign-language-prerecorded
https://www.w3.org/WAI/WCAG22/Techniques/general/G160

http://www.sign-lang.uni-hamburg.de/signingbooks/sbrc/grid/d71/guide13.htm
This is linked from the "Chapter 13, "Editing"" link on https://www.w3.org/WAI/WCAG22/Techniques/general/G54 and 
https://www.w3.org/WAI/WCAG22/Techniques/general/G81

https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/adaptivity-and-layout/
This is linked from the "Apple touch target size recommendations" link on https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced

https://rawgit.com/w3c/coga/master/gap-analysis/table.html
This is linked from the "User Needs Table 3: Entering data, error prevention & recovery" link on https://www.w3.org/WAI/WCAG22/Understanding/timeouts

https://rawgit.com/w3c/coga/master/issue-papers/privacy-security.html
This is linked from the "Security and Privacy Technologies issue paper from the Cognitive Task Force" link on https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced

https://w3c.github.io/coga/issue-papers/
This is linked from the "Cognitive Accessibility Issue Papers 4..." link on the following two pages:
https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum
https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced
It is also linked from the "Personalization and User Preferences" link on https://www.w3.org/WAI/WCAG22/Understanding/identify-purpose

https://w3c.github.io/coga/issue-papers/privacy-security.html
This is linked from the "Web Security and Privacy Technologies" link on the following two pages:
https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum
https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-enhanced

http://www.456bereastreet.com/archive/200504/about_fluid_and_fixed_width_layouts/
This is linked from the "About fluid and fixed width layouts" link on https://www.w3.org/WAI/WCAG22/Understanding/resize-text

https://www.msfw.com/Services/ContrastRatioCalculator
This is linked from the "Contrast Ratio Calculator" link on https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum and https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced

http://www.rnib.org.uk/travel-downloads-podcasts-and-apps?List=be9c76d3-7ad0-4e03-a1a0-e6f6953b8178&ID=16
This is linked from the "Overcoming the challenge of podcast transcription" link on https://www.w3.org/WAI/WCAG22/Understanding/audio-only-and-video-only-prerecorded

http://www.csail.mit.edu/
This is linked from the "MIT" link in the footer on https://www.w3.org/WAI/WCAG22/working-examples/failure-orientation-message/
Note that the content of this page is only visible in portrait layout.

https://www.ssa.gov/accessibility/bpl/bps/forms/timeout/default.htm
This is linked from the "SSA Best Practices Library: Timeouts" link on https://www.w3.org/WAI/WCAG22/Understanding/timeouts
